examples/cmd/snapshotting/Makefile

```diff
@@ -61,7 +61,7 @@ root-drive-with-ssh.img root-drive-ssh-key:
 	- mkdir temp
 	- git clone https://github.com/firecracker-microvm/firecracker temp
 	temp/tools/devtool build_rootfs
-	cp temp/build/rootfs/bionic.rootfs.ext4 root-drive-with-ssh.img
+	cp temp/build/rootfs/jammy.rootfs.ext4 root-drive-with-ssh.img
```

Comment:
do we need this hardcoded? won't the same issue occur the next time firecracker upgrades the Ubuntu version for building rootfs w/ devtool? not pretty, but could we just cp `*.rootfs.ext4` since the devtool only makes 1 rootfs?

Reply:
Yeah, make sense. 
I do have a comment message above mentioning we need to change this if firecracker uses different Ubuntu distro.
I can change it the `*.rootfs.ext4`, previously I am not sure if there are other rootfs.